fyne does not work, crashes

```
$ ./godcr  --mode=fyne
Launching desktop app with fyne
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xc3b87b]

goroutine 11 [running]:
fyne.io/fyne/widget.(*scrollRenderer).Layout(0xc00033a090, 0xb96, 0x6a9)
        /home/x/go/pkg/mod/fyne.io/fyne@v0.0.0-20190411071008-b3687258b083/widget/scroller.go:56 +0x3b
fyne.io/fyne/widget.(*baseWidget).resize(0xc00033e0a0, 0xb96, 0x6a9, 0x1469ce0, 0xc00033e0a0)
        /home/x/go/pkg/mod/fyne.io/fyne@v0.0.0-20190411071008-b3687258b083/widget/widget.go:28 +0x73
fyne.io/fyne/widget.(*ScrollContainer).Resize(0xc00033e0a0, 0xb96, 0x6a9)
        /home/x/go/pkg/mod/fyne.io/fyne@v0.0.0-20190411071008-b3687258b083/widget/scroller.go:117 +0x52
fyne.io/fyne/layout.(*fixedGridLayout).Layout(0xc00034c110, 0xc00009f6e0, 0x1, 0x1, 0xb96, 0x6a9)
        /home/x/go/pkg/mod/fyne.io/fyne@v0.0.0-20190411071008-b3687258b083/layout/fixedgridlayout.go:26 +0xd8
fyne.io/fyne.(*Container).layout(0xc00033e0f0)
        /home/x/go/pkg/mod/fyne.io/fyne@v0.0.0-20190411071008-b3687258b083/container.go:16 +0x11c
fyne.io/fyne.NewContainerWithLayout(0x144dd80, 0xc00034c110, 0xc00009f6e0, 0x1, 0x1, 0xc000089e00)
        /home/x/go/pkg/mod/fyne.io/fyne@v0.0.0-20190411071008-b3687258b083/container.go:111 +0xb6
github.com/raedahgroup/godcr/fyne.(*fyneApp).resizeScrollableContainer(0xc00013c2d0)
        /home/x/code/godcr/fyne/resize.go:57 +0x17b
github.com/raedahgroup/godcr/fyne.(*fyneApp).listenForWindowResizeEvents.func1(0xc0000a8a00, 0xc00013c2d0)
        /home/x/code/godcr/fyne/resize.go:35 +0xa1
created by github.com/raedahgroup/godcr/fyne.(*fyneApp).listenForWindowResizeEvents
        /home/x/code/godcr/fyne/resize.go:27 +0xd6

```

--- a/fyne/resize.go
+++ b/fyne/resize.go
@@ -46,6 +46,11 @@ func (app *fyneApp) listenForWindowResizeEvents() {
 // The idea is, if the content size is bigger than the maximum space available,
 // scroll bars become visible and more of the content can be seen by scrolling.
 func (app *fyneApp) resizeScrollableContainer() {
+	if app.pageContent == nil {
+		// only resize if page content has been prepared
+		return
+	}
+
 	// calculate the maximum available width and height to use for scroll container
 	windowSize := app.mainWindow.Content().Size()
 	scrollAreaWidth := windowSize.Width - menuSectionWidth - menuSectionPageSectionSeparation

